### PR TITLE
GraphGPS example

### PR DESCRIPTION
**Pull request by rusty1s**

### CODE REVIEW

**Review by wsad1 — APPROVED** [review]
Looks good. Thanks.

File: examples/graph_gps.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        for _ in range(num_layers):
+            nn = Sequential(
+                Linear(channels, channels),
+                ReLU(),
+                Linear(channels, channels),
+            )
+            conv = GPSConv(channels, GINEConv(nn), heads=4, attn_dropout=0.5)
+            self.convs.append(conv)
+
+        self.classifier = Linear(channels, 1)
```
[the remark this answers is not in the document]
Comment (rusty1s) [reply examples/graph_gps.py:40]: Oh, right. It's not a classification task at all, thanks for the catch.

**Comment by zhoulikeda** [comment]
I'have run the code, it seems cannot get a good result for ZINC dataset

**Comment by rusty1s** [comment]
Yes, it's not fully correct yet. Need to look into it.

**Pull request merged**